src/applications/mhv/medications/components/shared/ExtraDetails.jsx

```diff
@@ -88,6 +99,7 @@ const ExtraDetails = rx => {
 ExtraDetails.propTypes = {
   rx: PropTypes.shape({
     refillStatus: PropTypes.string,
+    dispStatus: PropTypes.string,
   }),
 };
```

Comment:
Im wondering if we still need to use the `refillStatus` field? doesn't `dispStatus` give enough information for the logic to work? It was my understanding that both of these fields serve the same purpose but the dipStatus gives a more accurate description and is sentence cased.  Or am I missing something? Let me know what you think.

Reply:
changed